ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/ProtocolScheduleBuilder.java

```diff
@@ -46,11 +46,13 @@ public ProtocolScheduleBuilder(
     validateForkOrdering();
 
     addProtocolSpec(
-        protocolSchedule, OptionalLong.of(0), MainnetProtocolSpecs.frontierDefinition());
+        protocolSchedule,
+        OptionalLong.of(0),
+        MainnetProtocolSpecs.frontierDefinition(config.getContractSizeLimit()));
```

Comment:
nit: perhaps extract config.getContractSizeLimit() to a local var to make the code little more readable. It's used quite a few times in this function.